qa/suites/upgrade/client-upgrade-mimic/mimic-client-x/basic/2-workload/devstack-tempest-gate.yaml

```diff
@@ -0,0 +1,56 @@
+tasks:
+- sequential:
+  - upgrade_workload
+- ceph: 
+- print: "**** done ceph"
+- qemu:
+    all:
```

Comment:
```suggestion
    client.1:
```

Reply:
fixed that and running `--filter "devstack-tempest-gate.yaml"` only

http://pulpito.front.sepia.ceph.com:80/yuriw-2019-04-30_20:15:26-upgrade:client-upgrade-mimic-nautilus-distro-basic-smithi/

Reply:
need to fix `need more than 0 values to unpack` @dillaman

Reply:
Sorry, yes -- for "basic" test group only "client.0" is defined.